etc/tempesta_fw.conf

```diff
@@ -1,1528 +1,70 @@
-#       hash <HASH_STRING> <CONNECTIONS_PER_SEC> <HTTP_REQUESTS_PER_SECOND>;
-#       ...
-#       hash <HASH_STRING> <CONNECTIONS_PER_SEC> <HTTP_REQUESTS_PER_SECOND>;
-#   }
-#
-#
-# STORAGE_SIZE is the size of the storage holding ja5h hashes to be monitored
-# by filtering code. Hashes are evicted by LRU algorithm. The value MUST be multiple
-# of 2^21. Defalut: 25 * 2^21
-#
-# HASH_STRING is a string value of a ja5h hash calculated from the HTTP request.
-# You can find these values in the access log or in ClickHouse-based 
-# analytics.
-#
-# CONNECTIONS_PER_SEC is a number of allowed connections per second for
-# clients identified by HASH_STRING.
-#
-# HTTP_REQUESTS_PER_SECOND is a number of allowed HTTP requests per second for
-# the clients identified by HASH_STRING.
-#
-# Examples:
-#   ja5h storage_size=2097152 {
-#       hash deadbeef12345678 10 1000;
-#       ...
-#       hash 1234abcdeeaabbcc 0 0;
-#   }
-#
-# Default:
-#   No HTTP filtering applied.
-#
+##
+# Recommended basic configuration.
+##
+
+listen 80;
+
+# TLS settings: uncomment for TLS connections
+# https://tempesta-tech.com/knowledge-base/Handling-clients/#listening-address
+#listen 443 proto=h2,https;
+#tls_certificate <absolute_path_to_cert>;
+#tls_certificate_key <absolute_path_to_key>;
+
+# Cache settings: uncomment to enable cache to all responses for GET and HEAD methods
+# https://github.com/tempesta-tech/tempesta/wiki/Caching-Responses
+#cache 2;
+#cache_fulfill * *;
+#cache_methods GET HEAD;
+#cache_ttl 3600;
+
+# PURGE settings: uncomment to allow cache purging
+# https://tempesta-tech.com/knowledge-base/Caching-Responses/#manual-cache-purging
+#cache_purge;
+#cache_purge_acl <IP_address>;
+
+# Logging settings: uncomment to enable access log in clickhouse
+# https://tempesta-tech.com/knowledge-base/Handling-clients/#access-log
+# and https://tempesta-tech.com/knowledge-base/TFW-Logger/
+#access_log mmap logger_config=<absolute_path_to_config>;
+
+# https://tempesta-tech.com/knowledge-base/Handling-clients/#error-responses
+block_action error reply;
+block_action attack drop;
+
+# Security settings: https://tempesta-tech.com/knowledge-base/HTTP-security/
+frang_limits {
+  request_rate 200;
+  request_burst 25;
+  tcp_connection_rate 100;
+  tcp_connection_burst 10;
+  concurrent_tcp_connections 500;
+  http_methods head post put get delete;
+}
+
+# Allow only following characters in URI: $%()+,/a-zA-Z0-9&?:-.[]_=
+# https://tempesta-tech.com/knowledge-base/HTTP-security/#custom-character-sets
+http_uri_brange 0x24-0x26 0x28-0x29 0x2b 0x2c 0x2f 0x41-0x5a 0x61-0x7a 0x30-0x39 0x3f 0x3a 0x2d 0x2e 0x5b 0x5d 0x5f 0x3d;
+
+# https://ddos-test.com: The system keeps idle TCP client connections open for more than 45.0 seconds.
+# https://tempesta-tech.com/knowledge-base/Handling-clients/#keep-alive-timeout
+keepalive_timeout 30;
+
+# https://tempesta-tech.com/knowledge-base/Backend-servers/
+srv_group srv_grp_1 { server <server_IP>:<server_port> conns_n=64; }
```

Comment:
This line won't be loaded by Tempesta FW, should be commented out. Probably add some mark to make it visible as the most crucial setting to define